qmlti/lib.php

```diff
@@ -408,6 +467,25 @@ function get_access_administrator($username) {
 
   }
 
+/*
+ * SOAP call to get an assessment's details 
+ *
+ *   returns the assessment or FALSE
+ */
+  function get_assessment($assessment_id) {
```

Comment:
This function is essentially a wrapper for the PerceptionSoap function of the same name.